pkg/reconciler/openshift/tektonpipeline/extension.go

```diff
@@ -197,7 +197,7 @@ func (oe openshiftExtension) Finalize(ctx context.Context, comp v1alpha1.TektonC
 		return err
 	}
 	if err := deleteInstallerSet(ctx, oe.operatorClientSet, pipeline,
-		postPipelineInstallerSet, postReconcileSelector); err != nil {
+		postPipelineInstallerSet, preReconcileSelector); err != nil {
```

Comment:
```suggestion
		preReconcileSelector, preReconcileSelector); err != nil {
```
the first one is an obsolete `component` argument. it willl work as it is, but it could be confusing for the future reader.

Reply:
yep. updated